templates/bootstrap/features.tmpl

```diff
@@ -29,7 +29,7 @@
             <div class="panel panel-default">
               <div class="panel-heading">
                 <h4 class="panel-title">
-                <% var scenarioTagsOnly = feature.tags ? _.drop(element.tags, feature.tags.length) :  element.tags %>
+                <% var scenarioTagsOnly = element.tags %>
```

Comment:
Will revert this change as this was more for a specific use where in if the tags are burried under the feature, they weren't visibile unless the feature twiste is expanded.